wormhole-connect/src/hooks/useAvailableWallets.ts

```diff
@@ -6,27 +6,19 @@ import { Chain } from '@wormhole-foundation/sdk';
 import config from 'config';
 import { WalletData, getWalletOptions } from 'utils/wallet';
 
-type GetWalletsLoading = {
-  state: 'loading';
+type walletOptions = {
+  state: 'result' | 'loading' | 'error';
+  options?: WalletData[];
+  error?: string;
 };
-type GetWalletsError = {
-  state: 'error';
-  error: string;
-};
-type GetWalletsResult = {
-  state: 'result';
-  options: WalletData[];
-};
-
-type GetWallets = GetWalletsLoading | GetWalletsError | GetWalletsResult;
```

Comment:
There is no need to have multiple wallet options type and to combine them. This type is local and we can have just one result type.